### PR TITLE
DOC-818 K8s Galactica release changes

### PR DESCRIPTION
**Pull request by kaitlynmichael**
DOC-818 
updated supported distros

### CODE REVIEW

**Review by rrelledge — APPROVED** [review]

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+Title: Redis Enterprise for Kubernetes Release Notes 6.0.20-<tbd> (Sept 2021)
+linktitle: 6.2.4-<tbd> (Sept 2021)
+description: Release notes for version 6.2.4-<tbd>
+weight: 67
+alwaysopen: false
+categories: ["Platforms"]
+---
+## Overview
+
+The Redis Enterprise [K8s 6.2.4-55](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.4-55) release provides support for the [Redis Enterprise Software release 6.2.4]({{<relref ""/rs/release-notes/rs-6-2-4-august-2021/>}}) and includes several enhancements and bug fixes.
```
Comment (kaitlynmichael) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:11]:
```suggestion
The Redis Enterprise [K8s 6.2.4-55](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.4-55) release provides support for the [Redis Enterprise Software release 6.2.4]({{<relref "/rs/release-notes/rs-6-2-4-august-2021/">}}) and includes several enhancements and bug fixes.
```

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+The Redis Enterprise [K8s 6.2.4-55](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.4-55) release provides support for the [Redis Enterprise Software release 6.2.4]({{<relref "/rs/release-notes/rs-6-2-4-august-2021/">}}) and includes several enhancements and bug fixes.
+
+The key new features, bug fixes, and known limitations are described below.
+
+## Images
+
+This release includes the following container images:
+
+* **Redis Enterprise**: `redislabs/redis:6.2.4-55` or  redislabs/redis:6.2.4-55.rhel7-openshift`
```
Comment (kaitlynmichael) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:19]:
```suggestion
* **Redis Enterprise**: `redislabs/redis:6.2.4-55` or  `redislabs/redis:6.2.4-55.rhel7-openshift`
```

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+Title: Redis Enterprise for Kubernetes Release Notes 6.0.20-<tbd> (Sept 2021)
+linktitle: 6.2.4-<tbd> (Sept 2021)
+description: Release notes for version 6.2.4-<tbd>
+weight: 67
+alwaysopen: false
+categories: ["Platforms"]
+---
+## Overview
+
+The Redis Enterprise [K8s 6.2.4-55](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.4-55) release provides support for the [Redis Enterprise Software release 6.2.4]({{<relref "/rs/release-notes/rs-6-2-4-august-2021/">}}) and includes several enhancements and bug fixes.
```
Comment (kaitlynmichael) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:11]:
```suggestion
The Redis Enterprise [K8s 6.2.4-55](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.4-55) release provides support for the [Redis Enterprise Software release 6.2.4]({{<relref "/rs/release-notes/rs-6-2-4-august-2021.md">}}) and includes several enhancements and bug fixes.
```

**Review by yuvallevy2 — APPROVED** [review]

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+* Support OpenShift 4.8 (K8s 1.21) (RED-59424)
+* Support K8s 1.21 - GKE (RED-59048)
+* Support K8s 1.21 - kOps (RED-59047)
+* Support K8s 1.19-1.21 - EKS (RED-60287)
+* Support K8s 1.19, 1.20 - AKS (RED-59050)
+* Support K8s 1.20 - Rancher (RED-59049)
+
+## Fixed bugs
+
+* Fixed issue with RS pods not recovering from container fails (RED-)
```
Comment (rrelledge) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:39]: Missing RED- ticket number?

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+Kubernetes-based 5.4.10 deployments seem to negatively impact existing 5.4.6 deployments that share a Kubernetes cluster.
+
+### Node CPU usage is reported instead of pod CPU usage (RED-36884)
+
+In Kubernetes, the node CPU usage we report on is the usage of the Kubernetes worker node hosting the REC pod.
+
+### Clusters must be named "rec" in OLM-based deployments (RED-39825)
+
+In OLM-deployed operators, the deployment of the cluster will fail if the name is not "rec". When the operator is deployed via the OLM, the security context constraints (scc) is bound to a specific service account name (i.e., "rec"). The workaround is to name the cluster "rec".
```
Comment (rrelledge) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:88]:
```suggestion
In OLM-deployed operators, the deployment of the cluster will fail if the name is not "rec". When the operator is deployed via the OLM, the security context constraints (scc) are bound to a specific service account name (i.e., "rec"). The workaround is to name the cluster "rec".
```

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+### REC extraLabels are not applied to PVCs on K8s versions 1.15 or older (RED-51921)
+
+In K8s 1.15 or older, the PVC labels come from the match selectors and not the
+PVC templates. As such, these versions can not support PVC labels. If this feature
+is required, the only fix is to upgrade the K8s cluster to a newer version.
+
+### REC might report error states on initial startup (RED-61707)
+
+There is not workaround at this time except to ignore the errors.
```
Comment (rrelledge) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:109]:
```suggestion
There is no workaround at this time except to ignore the errors.
```

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+Title: Redis Enterprise for Kubernetes Release Notes 6.0.20-<tbd> (Sept 2021)
+linktitle: 6.2.4-<tbd> (Sept 2021)
+description: Release notes for version 6.2.4-<tbd>
+weight: 67
+alwaysopen: false
+categories: ["Platforms"]
+---
+## Overview
+
+The Redis Enterprise [K8s 6.2.4-55](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.4-55) release provides support for the [Redis Enterprise Software release 6.2.4]({{<relref "/rs/release-notes/rs-6-2-4-august-2021.md">}}) and includes several enhancements and bug fixes.
```
Comment (rrelledge) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:11]: The K8s 6.2.4-55 link currently gives me a 404, but maybe this is just a GitHub permissions issue on my part though?

**Review by rrelledge — APPROVED** [review]
I left a few small suggestions.

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md (hunk 7 above)
Comment (amiramm) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:11]:
```suggestion
The Redis Enterprise [K8s 6.2.4-55](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.4-1) release provides support for the [Redis Enterprise Software release 6.2.4]({{<relref "/rs/release-notes/rs-6-2-4-august-2021.md">}}) and includes several enhancements and bug fixes.
```
@kaitlynmichael @rrelledge I need to tag the release. likely late on 9/14.

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+The key new features, bug fixes, and known limitations are described below.
+
+## Images
+
+This release includes the following container images:
+
+* **Redis Enterprise**: `redislabs/redis:6.2.4-55` or  `redislabs/redis:6.2.4-55.rhel7-openshift`
+* **Operator**: `redislabs/operator:6.2.4-<tbd>`
```
Comment (amiramm) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:20]:
```suggestion
* **Operator**: `redislabs/operator:6.2.4-1`
```

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+* **Services Rigger**: `redislabs/k8s-controller:6.0.20-<tbd>` or `redislabs/services-manager:6.2.4-<tbd>` (on the Red Hat registry)
```
Comment (amiramm) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:21]:
```suggestion
* **Services Rigger**: `redislabs/k8s-controller:6.0.20-1` or `redislabs/services-manager:6.2.4-1` (on the Red Hat registry)
```

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md (hunk 4 above)
Comment (amiramm) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:39]:
```suggestion
* Fixed issue with RS pods not recovering from container failures (RED-53042)
```

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+* Support K8s 1.21 - GKE (RED-59048)
+* Support K8s 1.21 - kOps (RED-59047)
+* Support K8s 1.19-1.21 - EKS (RED-60287)
+* Support K8s 1.19, 1.20 - AKS (RED-59050)
+* Support K8s 1.20 - Rancher (RED-59049)
+
+## Fixed bugs
+
+* Fixed issue with RS pods not recovering from container fails (RED-)
+* Fixed rare problem of RS pod restarting too early while statefulSet was rolling out, causing quorum loss (RED-53042)
```
Comment (amiramm) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:40]:
```suggestion
* Fixed rare problem of Redis Enterprise pod restarting too early while statefulSet was rolling out, causing quorum loss (RED-53042)
```

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+
+### No longer supported
+
+* GKE K8s version 1.17 (previously deprecated) 
+* kOps K8s version 1.15 (previously deprecated) 
+
+## Deprecation notice
+
+* kOps 1.16 and 1.17 are deprecated
+* VMWare TKGIE 1.7 (K8s 1.16), VMWare TKGIE 1.8 (K8s 1.17) are deprecated (no longer supported by VMWare)
```
Comment (amiramm) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:135]: @kaitlynmichael additional deprecation
```suggestion
* VMWare TKGIE 1.7 (K8s 1.16), VMWare TKGIE 1.8 (K8s 1.17) are deprecated (no longer supported by VMWare)
* Openshift 3.11 (K8s 1.11) is now deprecated. Redis will continue to support existing deployments for the lifetime of Openshift 3.11, but new deployments are strongly discouraged.
```

**Review by amiramm — CHANGES_REQUESTED** [review]
@kaitlynmichael thanks so much for putting this together - had some minor adjustments to share.

File: content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+* **Operator**: `redislabs/operator:6.2.4-<tbd>`
+* **Services Rigger**: `redislabs/k8s-controller:6.0.20-<tbd>` or `redislabs/services-manager:6.2.4-<tbd>` (on the Red Hat registry)
+
+## New features
+
+* Internode encryption configuration through K8s custom resources (RED-59699, RED-60318)
+
+## Feature improvements
+
+* Support for addition attribute in REDB secret containing comma seperated list of service names (RED-48469)
```
Comment (amiramm) [inline content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md:29]:
```suggestion
* Support for addition attribute in REDB secret containing comma separated list of service names (RED-48469)
```

**Comment by laurentdroin** [comment]
Can we add a new limitation that says that the length of the REC name + the length of the namespace name must be <= 45 characters?
This comes from https://redislabs.atlassian.net/browse/RED-62222

**Pull request merged**